context.go

```diff
@@ -28,13 +29,25 @@ type Context struct {
 func NewContext(config *Config) (*Context, error) {
 	var context Context
 	var ret C.int32_t
+	var err *C.tiledb_error_t
```

Comment:
This should probably be renamed something like `tdbErr` so that it doesn’t alias `err` (causing the trouble below).